packages/apps/kadena-docs/next.config.js

```diff
@@ -83,6 +85,7 @@ const withMDX = mdx({
 const nextConfig = {
   pageExtensions: ['ts', 'tsx', 'js', 'jsx', 'md', 'mdx'],
```

Comment:
Since this is a separate concern from the contents of this PR, I won't change it here, but this is good to note! @sstraatemans would you want to change this in the docs site?